app/src/graphql/operations.graphql

```diff
@@ -35,7 +36,24 @@ subscription CurrentGame($joinCode: String!) {
       id
       value
     }
-    cards {
+    cards(
+      where: {
+        _or: [
+          {
+            _and: [
+              { game: { screen_cards: { _eq: true } } }
+              { is_allowed: { _eq: true } }
+            ]
+          }
+          {
+            _and: [
+              { game: { screen_cards: { _eq: false } } }
+              { is_allowed: { _eq: null } }
+            ]
+          }
+        ]
+      }
+    ) {
```

Comment:
I think this should work @namoscato ? esp. since the OR clauses should are mutually exclusive

Reply:
Yeah, this looks sound to me. Would this also work?

```
game.screen_cards=false OR is_allowed=true
```

Reply:
Should unreviewed cards be incorporated in the `SubmissionCard.hasSimilarSubmissionError` logic?

Reply:
Thinking something like this might work?

```
_or: [
  { game: { screen_cards: { _eq: false } } }
  { is_allowed: { _neq: false } }
]
```

Having issues getting that going locally but have gotten a bit rusty with this stack again 😬